test/e2e/common/downward_api.go

```diff
@@ -183,6 +183,82 @@ var _ = framework.KubeDescribe("Downward API", func() {
 
 		testDownwardAPIUsingPod(f, pod, env, expectations)
 	})
+
+})
+
+var _ = framework.KubeDescribe("Downward API [Serial] [Disruptive] [Feature:LocalStorageCapacityIsolation]", func() {
+	f := framework.NewDefaultFramework("downward-api")
+
+	Context("Downward API tests for local ephemeral storage", func() {
+		BeforeEach(func() {
+			framework.SkipUnlessLocalEphemeralStorageEnabled()
+		})
+
+		It("should provide container's limits.ephemeral-storage and requests.ephemeral-storage as env vars [Feature:LocalStorageCapacityIsolation]", func() {
+			podName := "downward-api-" + string(uuid.NewUUID())
+			env := []v1.EnvVar{
+				{
+					Name: "EPHEMERAL_STORAGE_LIMIT",
+					ValueFrom: &v1.EnvVarSource{
+						ResourceFieldRef: &v1.ResourceFieldSelector{
+							Resource: "limits.ephemeral-storage",
+						},
+					},
+				},
+				{
+					Name: "EPHEMERAL_STORAGE_REQUEST",
+					ValueFrom: &v1.EnvVarSource{
+						ResourceFieldRef: &v1.ResourceFieldSelector{
+							Resource: "requests.ephemeral-storage",
+						},
+					},
+				},
+			}
+			expectations := []string{
+				fmt.Sprintf("EPHEMERAL_STORAGE_LIMIT=64*1024*1024"),
```

Comment:
why use `fmt.Sprintf` here and below?

Reply:
sorry, copy from the exsiting code directly

Reply:
Modified